controllers/courses.js

```diff
@@ -163,7 +164,7 @@ const editCourseHandler = (req, res, next) => {
 
 	Promise.all([
 		coursePromise,
-		classesAndGroupsPromise,
+		FEATURE_GROUPS_IN_COURSE_ENABLED ? classesAndGroupsPromise : classesPromise,
```

Comment:
Did you tested the turning the feature flag on and off?

Reply:
yes